<launch>
    <node pkg="nuice_abstraction" type="probe_control.py" name="probe_control"/>

    <node pkg="nuice_abstraction" type="basic_stepper.py" name="cat_stepper_abstract">
		<param name="stepper_name" value="/central_board/cat_stp"/>
		<param name="steps_per_unit" value="1"/>
		<param name="max" value="800"/>
		<param name="min" value="0"/>
		<param name="reverse" value="False"/>
		<param name="max_speed" value="400"/>
		<param name="accel" value="800"/>
		<param name="home_speed" value="200"/>
    </node>
</launch>